components/builder-web/stylesheets/_mixins.scss

```diff
@@ -24,7 +24,7 @@
   right: $small-spacing;
   height: rem(24);
   width: rem(24);
-  top: 46%;
+  top: 50%;
   transform: translateY(-50%);
```

Comment:
This does in fact center things vertically, however this Montserrat font causes things to appear slightly off, visually. This css is correct, let's keep it and plan to change fonts in the near future. These Montserrat fonts simply aren't well suited for the web.